### PR TITLE
Allow building without a device connected to adb

### PR DESCRIPTION
**Pull request by mariogrip**
This is usefull for halium since ssh is the perfered method there insted of adb

### CODE REVIEW

File: crossbuilder [hunk 1]
```diff
@@ -778,11 +779,18 @@ if [ -z "$COMMAND" ] ; then
     detect_package
     check_changelog_exists
     variables
-    echo "${POSITIVE_COLOR}Building $PACKAGE for $TARGET_ARCH and deploying to device.${NC}"
+    if [ "$DEPLOY" ] ; then
+        echo "${POSITIVE_COLOR}Building $PACKAGE for $TARGET_ARCH and deploying to device.${NC}"
+    else
+        echo "Could not find a device, will not deploy"
+        echo "${POSITIVE_COLOR}Building $PACKAGE for $TARGET_ARCH"
```
Comment (mardy) [inline crossbuilder]: Miss `${NC}` at the end of this line (to clear the color).

File: crossbuilder [hunk 2]
```diff
@@ -778,11 +779,18 @@ if [ -z "$COMMAND" ] ; then
     detect_package
     check_changelog_exists
     variables
-    echo "${POSITIVE_COLOR}Building $PACKAGE for $TARGET_ARCH and deploying to device.${NC}"
+    if [ "$DEPLOY" ] ; then
```
Comment (mardy) [inline crossbuilder]: remove the space before the `;`

Comment (mariogrip) [reply crossbuilder]: I just wanted to keep it consistent with the other if's, see above.

Comment (mardy) [reply crossbuilder]: Ah! Then ignore this :-)

**Pull request merged**